lamindb_setup/core/_settings_store.py

```diff
@@ -65,6 +65,7 @@ class UserSettingsStore(BaseSettings):
     email: str
     password: str
     access_token: str
+    api_key: Optional[str] = None
```

Comment:
@falexwolf this is actually enough to avoid the validation error.
Do you think the rewrite of settings load is still needed? It is not hard to do, especially via `dotenv_values` like [here](https://github.com/laminlabs/lamindb-setup/blob/bc17873eb9b8ece81f3bbea88448456b88d9b551/lamindb_setup/core/_settings_storage.py#L204), i just think it is reasonable to offload loading and validation logic to `pydantic-settings`.

Reply:
Ok! If users don't run into errors that's alright!